tag-historian/nitaghistorian-cloud.yml

```diff
@@ -114,16 +114,12 @@ definitions:
 
           - modifyHistoricalTagValues: The ability to modify properties of historical tag values
 
-          - deleteHistoricalTagValues: The ability to delete historical tag values
-
         type: object
         properties:
           queryHistoricalTagValues:
             $ref: '#/definitions/Operation'
           modifyHistoricalTagValues:
             $ref: '#/definitions/Operation'
-          deleteHistoricalTagValues:
```

Comment:
Thoughts on removing this from the response returned by the service as well? Feels kinda low ROI to me.

Reply:
I wouldn't worry about it.